crates/bevy_math/src/bounding/bounded3d/primitive_impls.rs

```diff
@@ -21,7 +21,7 @@ impl Bounded3d for Sphere {
     }
 }
 
-impl Bounded3d for Plane3d {
+impl Bounded3d for InfinitePlane3d {
```

Comment:
This seems wrong. Why would an *infinite* plane have bounds?

Reply:
Semantically I agree, but it can still be useful to be able to compute the AABB for collision detection purposes. You would still want planes (and half-spaces) to be included in BVHs for example. Or you could have a `Shape3d` trait that requires `Bounded3d`.

For reference, Parry's [`HalfSpace` has AABB methods](https://github.com/dimforge/parry/blob/8823198d704a85b748b8c45ee2f54e35f24a080e/src/bounding_volume/aabb_halfspace.rs)

Reply:
I'll grant that, and it makes more sense looking at the actual impl. I consider this resolved.